lib/service/invoices.go

```diff
@@ -190,7 +190,8 @@ func (svc *LndhubService) PayInvoice(ctx context.Context, invoice *models.Invoic
 
 	// The payment was successful.
```

Comment:
```suggestion
	// The payment was successful.
	// these changes to the invoice are persisted in the `HandleSuccessfulPayment` function
```

To make it clear that the saving happens in the function.